### PR TITLE
SGCluster:set ingress rules to match Scylla ports

### PR DESCRIPTION
**Pull request by yaronkaikov**
Currently SGCluster is open inside for all protocols and ports. This
security group is being used by Scylla internal cluster communication,
let's make sure we only enable relevant tcp ports based on https://docs.scylladb.com/operating-scylla/admin/#id6

Fixes: https://github.com/scylladb/scylla-machine-image/issues/213

### CODE REVIEW

**Comment by benipeled** [comment]
Please enable ingress ICMP as well by:
```yaml
- IpProtocol: icmp
  FromPort: 8
  ToPort: -1
  CidrIp: !Ref CIDR
```
In addition:
-  I see no reason for the `SGUser` section - please double check and remove it.
- the `SecurityGroupEgress` is duplicated (`SGCluster` & `SGAdmin`) we might want to take it out for a dedicated one `SGExternal` - WDYT?

**Comment by yaronkaikov** [comment]
> Please enable ingress ICMP as well by:
> 
> ```yaml
> - IpProtocol: icmp
>   FromPort: 8
>   ToPort: -1
>   CidrIp: !Ref CIDR
> ```
> 
> In addition:
> 
> * I see no reason for the `SGUser` section - please double check and remove it.
> * the `SecurityGroupEgress` is duplicated (`SGCluster` & `SGAdmin`) we might want to take it out for a dedicated one `SGExternal` - WDYT?

It looks like `SGUser` is not being used, so i will remove it.

**Comment by yaronkaikov** [comment]
@benipeled regarding the `SecurityGroupEgress`, i think that for `SGCluster` we are currently setting it the wrong way. we should specify `CIDR` as we do for `SecurityGRoupIngress`

**Comment by benipeled** [comment]
> @benipeled regarding the `SecurityGroupEgress`, i think that for `SGCluster` we are currently setting it the wrong way. we should specify `CIDR` as we do for `SecurityGRoupIngress`

`SecurityGRoupIngress` is about inbound traffic so we use the CIDR provides by the user which is the user's private subnet,
but `SecurityGroupEgress` is about outbound traffic which I'm not sure we care about,
Maybe we can remove it completely but here again we complex the flow, if user will try to import something from internal (or any other outbound communication) he'll have to deal with network "issues" and may give up.

**Review by gcarmin — APPROVED** [review]
LGTM

File: aws/cloudformation/scylla.yaml.j2 [hunk 1]
```diff
@@ -297,8 +298,22 @@ Resources:
           Value: !Sub '${ScyllaClusterName}-RT'
         - Key: ScyllaClusterName
           Value: !Ref ScyllaClusterName
+      VpcId: !Ref VPC
 
+  SGExternal:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: Security group for the admin
```
Comment (benipeled) [inline aws/cloudformation/scylla.yaml.j2:306]: "Security group for outbound rules"

**Review by benipeled — APPROVED** [review]
Require one tiny fix but LGTM

File: aws/cloudformation/scylla.yaml.j2 (hunk 1 above)
Comment (yaronkaikov) [reply aws/cloudformation/scylla.yaml.j2]: Fixed,. Thanks

**Review by bentsi — APPROVED** [review]

**Pull request merged**